### ISSUE TITLE
Inflation of nft

### ISSUE THREAD

**Issue by ilNomade** [sunflower-land/sunflower-land#1068]
I would like to bring to light a topic concerning the nft t2 / t3 of crops and wood.
With the advancement of the halvings we will find ourselves with the nft of crops super inflated compared to those of wood, this leads to uselessness of them. Just think that nancy (t1 of crops) is currently worth 1/3 of woody (t1 of wood) and this proportion will continue to increase with the next halving. 

![Schermata 2022-06-09 alle 11 00 13](https://user-images.githubusercontent.com/106592209/172808773-5f9fe620-7087-4222-aaee-6671cb025bef.png)
![Schermata 2022-06-09 alle 11 00 23](https://user-images.githubusercontent.com/106592209/172808781-d9c081b9-26e7-4d44-8133-712550cb475c.png)

We need a rebalancing of the NFTs with a change of funcionality.

**Comment by future2back** [comment]
> you didn't understand the speech

Yes i do, this problem is already known but nothing has been proposed to resolve. Looks like NFTs are for a select group, people that have 3k+ SFL, 1k gold, eggs, chicken and enought resources to get it. These group dont care about this imbalance you mentioned. They're not playing that real, that's why they don't feel it. New NFTs will be released soon and they don't worry about it. They need to be more careful with the user experience, otherwise the economy will be compromised.

**Comment by Wolwar** [comment]
Game makers should not ignore such a problem when considering a long-term game, otherwise we will step into a troubled period with halvings, not fud, but facts.In short, it is a very important issue.

**Comment by adamhannigan** [comment]
Thanks @ilNomade for raising this.

We are currently addressing this in our game design for land expansion. We are well aware of the current problems with the game and making significant changes.

Could you please post this in our ideas portal instead. This gets more visibility from our community and helps voting and raising issues to the top. We try keep Github focussed on bug reports, technical discussions and art requests.

https://sunflower-land.ideas.aha.io/

Thanks!
